packages/graphql/src/translate/translate-top-level-match.ts

```diff
@@ -70,21 +74,24 @@ export function createMatchClause({
         matchQuery = new Cypher.db.FullTextQueryNodes(matchNode, indexName, phraseParam);
 
         const labelsChecks = node.getLabels(context).map((label) => {
-            return Cypher.in(new Cypher.Literal(label), Cypher.labels(matchNode));
+            return Cypher.in(new Cypher.Literal(label), Cypher.labels(matchNode as Cypher.NamedNode));
```

Comment:
Lets try to avoid casting :smiley:

Reply:
This was just because typescript wasn't picking up that it can't be undefined at this point. A small refactor has removed the need for this now though 😄